pontoon/pretranslation/pretranslate.py

```diff
@@ -58,6 +58,17 @@ def get_pretranslations(entity, locale, preserve_placeables=False):
 
         pretranslation = serializer.serialize_entry(entry)
 
+        # Parse and serialize pretranslation again in order to assure cannonical style
+        parsed_pretranslation = parser.parse_entry(pretranslation)
+
+        if isinstance(parsed_pretranslation, ast.Junk):
+            log.info(
+                f"Fluent pretranslation error: Invalid translation: {pretranslation}"
+            )
+            return []
```

Comment:
On a second thought, I'm not sure we should make this change:
- Pretranslation with an Error is still better than no pretranslation
- It's not really in scope of this feature (with change in this PR, no additional messages will become Junk)

Reply:
I disagree, because pretranslations are allowed to go to production, and we should not allow invalid messages to go to production. And I would say that this is well within "Properly serialize Fluent placeables generated by Pretranslation".

Reply:
Pretranslations with errors don't go to production.